stylus/components/button.styl

```diff
@@ -243,6 +215,7 @@ $button-client
     justify-content   center
     align-items       center
     height            auto
+    min-height        rem(56)
```

Comment:
Why are we adding this?

Reply:
To match the design. It should have been done from the beginning. I may be overzealous on this one.